### PR TITLE
Remove formatting rules

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: rules/eslint/es6.js [hunk 1]
```diff
@@ -9,15 +9,8 @@ module.exports = {
   },
   'rules': {
     // require braces in arrow function body
```
Comment (sallas) [inline rules/eslint/es6.js]: Remove left over comment.

File: rules/eslint/style.js [hunk 2]
```diff
@@ -3,31 +3,11 @@
 module.exports = {
   'rules': {
     // enforce spacing inside array brackets
-    'array-bracket-spacing': 0, // NOTE - DO NOT TOUCH. Use rule in babel.js
```
Comment (sallas) [inline rules/eslint/style.js]: 'NOTE - DO NOT TOUCH' Everything fine here?
Also another lonely lost comment.

**Comment by DanMeakin** [comment]
Thanks for the comments. I have removed the old comments (I thought I got all of them!). I think the "DO NOT TOUCH" stuff is fine, where I have also removed it from the Babel rules.

On versioning, I thought it would make sense to do a major version bump for this change. It is a fairly fundamental change to this repo as we are basically telling users to change their tooling. Thoughts?

**Comment by madsgodvinjensen** [comment]
Major bump sounds right.

**Pull request merged**